pkgs/applications/video/mpv/scripts/smart-copy-paste-2.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildLua,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  xclip,
+}:
+buildLua (finalAttrs: {
+  pname = "smart-copy-paste-2";
+  version = "0-unstable-2023-11-25";
+
+  scriptPath = "scripts/SmartCopyPaste_II.lua";
+  src = fetchFromGitHub {
+    owner = "Eisa01";
+    repo = "mpv-scripts";
+    rev = "48d68283cea47ff8e904decc9003b3abc3e2123e";
+    hash = "sha256-95CAKjBRELX2f7oWSHFWJnI0mikAoxhfUphe9k51Qf4=";
+  };
+
+  patchPhase = ''
```

```suggestion
  prePatch = ''
```
or postPatch. Otherwise you have to include runHook prePatch/postPatch